src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -268,7 +440,11 @@ bool operator==(const Translation<Dim>& lhs, const Translation<Dim>& rhs) {
          (lhs.f_of_r_ == nullptr) == (rhs.f_of_r_ == nullptr) and
          ((lhs.f_of_r_ == nullptr and rhs.f_of_r_ == nullptr) or
           *lhs.f_of_r_ == *rhs.f_of_r_) and
-         lhs.center_ == rhs.center_;
+         lhs.center_ == rhs.center_ and
+         (!lhs.inner_radius_.has_value() == !rhs.inner_radius_.has_value() or
```

Comment:
A `std::optional` actually does all this checking in its `operator==` so you just have to do `lhs.inner_radius_ == rhs.inner_radius_ and lhs.outer_radius_ == rhs.outer_radius_`

Reply:
does it also check the values inside?

Reply:
Yup!